src/components/BaseHeading.jsx

```diff
@@ -5,7 +5,7 @@ const BaseHeading = ({ level, text }) => {
   const HeadingTag = `h${level}`
 
   return (
-    <HeadingTag className="text-4xl text-center md:text-6xl">
+    <HeadingTag className="text-4xl text-center font-header md:text-6xl">
```

Comment:
Praise(nb): Good use of the `font-header` class you created! Now wherever we use `BaseHeading` we will get this font, and in addition we can use it in other places where it's appropriate.